README.md

```diff
@@ -161,61 +173,67 @@ and the various connector integrations under the `src/connector` sub-directory.
 
 <!-- ### Sub-Crates -->
 
-<!--
-| Crate | Stability | Master | Docs | Example |
-|--------|-----------|-------|:----:|:------:|
-| [masking](./crates/masking) | [![experimental](https://raster.shields.io/static/v1?label=&message=experimental&color=orange)](https://github.com/emersion/stability-badges#experimental) | [![Health](https://raster.shields.io/static/v1?label=&message=unknown&color=333)]() | [![docs.rs](https://raster.shields.io/static/v1?label=&message=docs&color=eee)](https://docs.rs/masking) | [![Open in Gitpod](https://raster.shields.io/static/v1?label=&message=try%20online&color=eee)]() |
-| [router](./crates/router) | [![experimental](https://raster.shields.io/static/v1?label=&message=experimental&color=orange)](https://github.com/emersion/stability-badges#experimental) | [![Health](https://raster.shields.io/static/v1?label=&message=unknown&color=333)]() | [![docs.rs](https://raster.shields.io/static/v1?label=&message=docs&color=eee)](https://docs.rs/router) | [![Open in Gitpod](https://raster.shields.io/static/v1?label=&message=try%20online&color=eee)]() |
--->
-
-### Files Tree Layout
+### 🌳 Files Tree Layout
 
 <!-- FIXME: this table should either be generated by a script or smoke test
 should be introduced, checking it agrees with the actual structure -->
 
 ```text
```

Comment:
Can we add a one-liner for each directory?

Reply:
Yes, captain!